File: Unidad 3/C#/P10_U3.cs
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int hrtrab, hrext;
            double pago, sueldosem = 0;
            Console.WriteLine("¿Cuantas horas trabajaste?");
            hrtrab = Convert.ToInt32(Convert.ToDouble(Console.ReadLine()));
            Console.WriteLine("¿Cuanto es el pago por hora?");
            pago = Convert.ToInt32(Convert.ToInt32(Console.ReadLine()));

            if (hrtrab <= 40 )
            {
                sueldosem = hrtrab * pago;
                Console.WriteLine("Recibiras un total de $" + sueldosem);
            }
            else
            {
                hrext = hrtrab - 40;
                sueldosem = 40 * pago + hrext * (pago * 2);
                Console.WriteLine("Por trabajar " + hrext + " horas extra, usted recibe un total de $" + sueldosem);


            }
        }
    }
}

File: Unidad 3/JAVA/P10_U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p10_u3;

import java.util.Scanner;

/**
 *
 * @author LaboratorioLTI2
 */
public class P10_U3 {

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int hrtrab, hrext;
            double pago, sueldosem = 0;
            System.out.println("¿Cuantas horas trabajaste?");
            hrtrab = sc.nextInt();
            System.out.println("¿Cuanto es el pago por hora?");
            pago = sc.nextInt();

            if (hrtrab <= 40 )
            {
                sueldosem = hrtrab * pago;
                System.out.println("Recibiras un total de $" + sueldosem);
            }
            else
            {
                hrext = hrtrab - 40;
                sueldosem = 40 * pago + hrext * (pago * 2);
                System.out.println("Por trabajar " + hrext + " horas extra, usted recibe un total de $" + sueldosem);
            }
    }
}
